systems/data-plan/package/pkg/db/package_repo_test.go

```diff
@@ -0,0 +1,219 @@
+package db
```

Comment:
Same comments as for the Base rates sub system  repo tests.

Reply:
addressed